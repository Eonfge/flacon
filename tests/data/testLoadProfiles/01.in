[Profiles/AAC]
    Format=AAC
    Name=Aac
    Bitrate=256
    Quality=100
    UseQuality=true
    BitsPerSample=16
    SampleRate=44100

[Profiles/FLAC]
    Format = FLAC
    Name = Flac
    Compression=5
    ReplayGain=Disable
    BitsPerSample=24
    SampleRate=48000


[Profiles/MP3]
    Format = MP3
    Name = Mp3
    Bitrate=320
    Preset=vbrStandardFast
    Quality=4
    ReplayGain=Disable
    BitsPerSample=16
    SampleRate=44100

[Profiles/OGG]
    Format = OGG
    Name = Ogg
    MaxBitrate=
    MinBitrate=
    NormBitrate=
    Quality=7
    ReplayGain=Disable
    UseQuality=true
    BitsPerSample=16
    SampleRate=44100

[Profiles/OPUS]
    Name = Opus
    Format = OPUS
    Bitrate=96
    BitrateType=VBR
    BitsPerSample=0
    SampleRate=0

[Profiles/WV]
    Name = WV
    Format = WV
    Compression=1
    ReplayGain=Disable
    BitsPerSample=32
    SampleRate=9600

[Profiles/WAV]
    Format=WAV
    Name=WAV
    BitsPerSample=0
    SampleRate=44100
