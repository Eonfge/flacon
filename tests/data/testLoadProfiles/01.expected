[Profiles/AAC]
    Format=AAC
    Name=Aac
    Bitrate=256
    Quality=100
    UseQuality=true

[Profiles/FLAC]
    Format = FLAC
    Name = Flac
    Compression=5
    ReplayGain=Disable

[Profiles/MP3]
    Format = MP3
    Name = Mp3
    Bitrate=320
    Preset=vbrStandardFast
    Quality=4
    ReplayGain=Disable

[Profiles/OGG]
    Format = OGG
    Name = Ogg
    MaxBitrate=
    MinBitrate=
    NormBitrate=
    Quality=7
    ReplayGain=Disable
    UseQuality=true

[Profiles/OPUS]
    Name = Opus
    Format = OPUS
    Bitrate=96
    BitrateType=VBR

[Profiles/WV]
    Name = WV
    Format = WV
    Compression=1
    ReplayGain=Disable
